web/src/store/index.ts

```diff
@@ -1,28 +1,44 @@
 import Vue from "vue";
 import Vuex from "vuex";
-import { ApiData, fetchData, Diff, Kmer, Metadata, File, ObjMap } from "@/api/api";
+import { ApiData, fetchData, Diff, Kmer, Metadata, File, ObjMap, Block, fetchBlocks } from "@/api/api";
 
 Vue.use(Vuex);
 
+// noinspection SillyAssignmentJS
```

Comment:
Is this still needed? I'm not so fond of editor-specific comments in the codebase.

Reply:
This was added by accident. I'll remove this